wlog/logger_provider_jsonmarshal.go

```diff
@@ -41,28 +41,28 @@ func (l *jsonMapLogger) Log(params ...Param) {
 func (l *jsonMapLogger) Debug(msg string, params ...Param) {
 	switch l.level {
 	case DebugLevel:
-		l.logOutput(append(params, StringParam("message", msg), StringParam("level", "DEBUG")))
+		l.logOutput(append([]Param{StringParam("message", msg), StringParam("level", "DEBUG")}, params...))
```

Comment:
Can you explain the reasoning that `params` becomes appended instead of prepended as part of this change?